lib/src/arg_parser.dart

```diff
@@ -2,6 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+// TODO(nweiz): Remove this ignore when sdk#30084 is fixed or when args no
```

Comment:
"kevmoo", unless you're unilaterally signing up Natalie for work. :)

Reply:
This was a copy-paste from analysis_options, so I left the owner as-is.